### ISSUE TITLE
[ios] Open the TrackPP on saving the Track Recording

### ISSUE THREAD

**Issue by kirylkaveryn** [organicmaps/organicmaps#9938]
Continues https://github.com/organicmaps/organicmaps/pull/9785

When the user saves the current Track Recording it would be natural to open the Track Place Page with the current track.

**Comment by pastk** [comment]
From my POV a tap on the TR widget / button should open tracks PP with buttons like "Stop", "Pause"...

After TR is stopped via this button its of course natural to keep the PP open, but it'll be changed from a "track being recorded" into a finished track PP.
WDYT?

**Comment by kirylkaveryn** [comment]
> From my POV a tap on the TR widget / button should open tracks PP with buttons like "Stop", "Pause"...

Yes. This is how I want to implement this feature. The alert will be totally removed.

> After TR is stopped via this button its of course natural to keep the PP open, but it'll be changed from a "track being recorded" into a finished track PP.

Yes. It is already implemented but I'm struggling withe a TR PP screen design...

![Simulator Screen Recording - iPhone 16 Pro - 2024-12-26 at 19 34 51](https://github.com/user-attachments/assets/fe81a5de-c7bc-4585-996a-e3fead2036d9)

**Comment by pastk** [comment]
The approach presented on the video looks very good!
Might use some polishing, e.g. rename the Save button to Stop..
